wagtailio/core/wagtail_hooks.py

```diff
@@ -2,37 +2,16 @@
 from django.core.files.storage import get_storage_class
 from django.shortcuts import redirect
 from django.utils.cache import add_never_cache_headers
-from django.utils.html import format_html, format_html_join
+from django.utils.html import format_html
 
-from wagtail.core import hooks
-from wagtail.core.whitelist import allow_without_attributes
+from wagtail import hooks
 from wagtail.documents import get_document_model
 from wagtail.documents.models import document_served
+from wagtail.whitelist import allow_without_attributes
 
 from storages.backends.s3boto3 import S3Boto3Storage
 
 
-def editor_js():
-    js_files = ["js/vendor/hallo-code.js"]
-    js_includes = format_html_join(
-        "\n",
-        '<script src="{0}{1}"></script>',
-        ((settings.STATIC_URL, filename) for filename in js_files),
-    )
-
-    return js_includes + format_html(
-        """
-        <script>
-            registerHalloPlugin('halloheadings');
-            registerHalloPlugin('hallocode');
-        </script>
-        """
-    )
-
-
-hooks.register("insert_editor_js", editor_js)
-
-
 def editor_css():
     return format_html(
         '<link rel="stylesheet" href="'
```

Comment:
might be nice to replace the raw font-awesome usage with https://github.com/allcaps/wagtail-font-awesome-svg